src/main.go

```diff
@@ -178,6 +235,9 @@ func main() {
 		log.Infoln("initialized system-metrics...", time.Since(p))
 		chainSet.Register(chain.Name, chain)
 
+		nullEncodedData := "0|0|0|0|0|0|0|0|0|0|0|0|0|0"
```

Comment:
This is a zero block. Null block means a block with value as `null`. Please make sure you go through the above mentioned null example in apexcharts before you make changes here.